Modules/LDAPObject.c

```diff
@@ -289,16 +294,27 @@ attrs_from_List( PyObject *attrlist, char***attrsp, PyObject** seq) {
                 LDAPerror_TypeError("expected bytes in list", item);
```

Comment:
As I was reading this, I also changed the comment “Encoded by Python to UTF-8” to “Encoded *in* Python to UTF-8”, since it's python-ldap's Python code doing the encoding.